src/dotnet/APIView/APIView/Model/V2/ReviewLine.cs

```diff
@@ -106,6 +106,7 @@ private string ToString(bool includeAllTokens)
             StringBuilder sb = new();
             foreach (var token in filterdTokens)
             {
+                sb.Append(token.HasPrefixSpace == true ? " " : string.Empty);
                 sb.Append(token.Value);
                 sb.Append(token.HasSuffixSpace == true ? " " : string.Empty);
```

Comment:
This logic needs to be smarter. If the previous token has "HasSuffixSpace" set to true, then this needs to be ignored so you don't end up with a double space.

Reply:
Done.